packages/clerk-js/src/core/tokenCache.ts

```diff
@@ -80,6 +80,10 @@ const MemoryTokenCache = (prefix = KEY_PREFIX): TokenCache => {
 
     entry.tokenResolver
       .then(newToken => {
+        if (!newToken.jwt) {
+          return deleteKey();
+        }
```

Comment:
Was this issue caused by an error thrown inside the `tokenResolver` promise?

Reply:
This change attempts to fix the case where the network is offline causing the `Token.create` to return a `newToken : null`. Currently an error related to `Cannot read properties of null (reading 'jwt')` is thrown due to code in the following lines. To mitigate that (with the code above) I've added this guard to delete the key (if it exists) and return since there is no cookie to set.
Does this make any sense?